src/scraper.ts

```
@@ -862,6 +863,16 @@ export class Scraper {
     );
   }
 
+  /**
+   * Delete a tweet with the given ID.
+   * @param tweetId The ID of the tweet to delete.
+   * @returns A promise that resolves when the tweet is deleted.
+   */
+  public async deleteTweet(tweetId: string): Promise<void> {
+    // Call the likeTweet function from tweets.ts
+    await deleteTweet(tweetId, this.auth)
```

ESLint

[error] Insert `;`

(prettier/prettier)